federatedscope/nlp/baseline/fedavg_lr_on_synthetic.yaml

```diff
@@ -10,7 +10,6 @@ federate:
 data:
   root: data/
   type: synthetic
-  batch_size: 64
```

Comment:
Maybe we should add `dataloader.batch_size` in this yaml file to keep it consistent with others

Reply:
As the default setting is `cfg.dataloader.batch_size = 64`, should we add `dataloader.batch_size` to this yaml file?